frontends/web/src/components/badge/badge.tsx

```diff
@@ -1,20 +1,15 @@
-import { Component} from 'react';
+import { FunctionComponent } from 'react';
 import style from './badge.module.css';
 
-interface BadgeProps {
+interface Props {
     type: string;
```

Comment:
Was used with `'generic' | 'primary' | 'secondary'`  in the past, but now it is only used with 'generic', bit useless, but ok for this PR. I think we'll remove this component once e do a settings revamp.